lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm

```diff
@@ -133,16 +133,22 @@ sub get_levels {
 
 sub determine_genome_build {
   my $self = shift;
-  my $ref_build_name1 = shift;
-  my $genome_build = "";
-  my $b37_build =
-    Genome::Model::Build::ReferenceSequence->get(name => "GRCh37-lite-build37");
-  my $ref_build1 = Genome::Model::Build::ReferenceSequence->get(name => $ref_build_name1);
-  if($b37_build->is_compatible_with($ref_build1)) {
-    return "b37";
+  my $supplied_build_name = shift;
+
+  my $canonical_b37_build = Genome::Model::Build::ReferenceSequence->get(name => "GRCh37-lite-build37");
+  my $canonical_b38_build = Genome::Model::Build::ReferenceSequence->get(name => "GRC-human-build38");
+  my $supplied_build = Genome::Model::Build::ReferenceSequence->get(name => $supplied_build_name);
+  my $igv_named_build = '';
+  if($canonical_b37_build->is_compatible_with($supplied_build)) {
+    $igv_named_build = "b37";
+  } elsif ($canonical_b38_build->is_compatible_with($supplied_build)){
+    $igv_named_build = "hg38";
+  }
```

Comment:
This could be simplified using `List::MoreUtils qw(each_array)`, which would make it easier to expand this list in the future. 
Untested code sample:
```perl 
use List::MoreUtils qw(each_array);
my @reference_build_names = ('GRCh37-lite-build37', 'GRC-human-build38');
my @igv_build_names = ('b37', 'hg38');
my $iterator = each_array(@reference_build_names, @igv_build_names);
my $supplied_build = Genome::Model::Build::ReferenceSequence->get(name => $supplied_build_name);
my $igv_named_build = '';
while (my ($reference_build_name, $igv_build_name) = $iterator->next()) {
    my $reference_build = Genome::Model::Build::ReferenceSequence->get(name => $reference_build_name)
    if ($reference_build->is_compatible_with($supplied_build)) {
        $igv_named_build = $igv_build_name;
        last;
    }
}
```